schedule.html

```diff
@@ -409,29 +409,29 @@ <h3 style="color: red;">Abstract</h3>
             </div>
           </div>
 
-          <div id="nardonCard" class="row align-items-center border-bottom py-5" style="width: 100%;">
+           <div id="darwinCard" class="row align-items-center border-bottom py-5" style="width: 100%;">
             <div class="row align-items-stretch">
               <div class="col-md-3 text-white mb-3 mb-md-0"><span class="h4">3:00</span> <span>PM</span></div>
               <div class="ml-auto">
                 <!-- The Modal -->
-                <div id="nardon" class="modal">
+                <div id="darwin" class="modal">
 
                   <!-- Modal content -->
                   <div class="modal-content">
                     <h3 style="color: red;">Abstract</h3>
                     <br/>
-                    <p>How Java developers can use their architecture skills and Java toolsets to solve Data Engineering and Machine Learning engineering problems to take Data Science projects to the next level.</p>
+                    <p>Want real cross-platform development from a single codebase? Does Java still fit that bill? Flutter is Google's new cross-platform toolkit that targets Android, iOS, Windows, Mac, Linux and the browser from a single codebase. Come and see it in action and judge for yourself.
+                    </p>
                   </div>
 
                 </div>
-                <div onclick="openAbstract('nardon')" title="Click for Abstract"><h2 class="text-white"><a href="#nardonCard" class="text-primary d-block mb-3" data-aos="fade-left" data-aos-delay="300">Data Engineering and Machine Learning Engineering for Java Developers</a></h2>
-                  <!--span class="popuptext" id="moraes">Abstract: <br/>They told you that using Java with containers were great, but they never mentioned that wouldn't be that easy, right? Luckily there are have some best practices that will rescue your application from failing. Come to this session and learn them.</span-->
+                <div onclick="openAbstract('darwin')" title="Click for Abstract"><h2 class="text-white"><a href="#darwinCard" class="text-primary d-block mb-3" data-aos="fade-right" data-aos-delay="300">Flutter is the new Java - or is it?</a></h2>
```

Comment:
Why is it a fade-right instead of a fade-left? Same below, it switches. Why would a card fade on left instead of right?

Reply:
Answering my question: I think it randomly currently fades left or right. Originally, it was probably ordered left, right, left, right. Doesn't matter.

Reply:
Thanks Henri. I didn't notice a pattern. Glad it turned out now to matter